Meter.c

```diff
@@ -120,40 +132,61 @@ static void BarMeterMode_draw(Meter* this, int x, int y, int w) {
    assert(startPos <= w);
    assert(startPos + w <= RichString_sizeVal(bar));
 
+   const Settings* settings = this->host->settings;
    int blockSizes[10];
 
    // First draw in the bar[] buffer...
    int offset = 0;
+   const wchar_t* barChar = &bars[settings->barType][1];
+   const uint8_t barLen = wcslen(barChar);
```

Comment:
No sense to downcast this local variable to `uint8_t`. Just use `size_t` for `barLen`.

Reply:
oh will do that